mxd/backend-service/src/main/java/org/eclipse/tractusx/mxd/backendservice/BackendServiceExtension.java

```diff
@@ -78,10 +78,8 @@ public class BackendServiceExtension implements ServiceExtension {
     @Inject
     private TypeManager typeManager;
 
-    @Inject(required = false)
     private ContentStatementsService statements;
 
-    @Inject(required = false)
     private TransferStatementsService transferStatementsService;
```

Comment:
Same comment as above.

Reply:
This injection has been removed already. It's red marked.